core/src/test/scala/simulacrum/pc.scala

```diff
@@ -0,0 +1,18 @@
+package simulacrum
+
+import org.scalatest.WordSpec
+import org.scalatest.Matchers._
+import org.scalatest.OptionValues._
+
+import org.ensime.pcplod._
+
+class PresentationCompilerTest extends WordSpec {
+  "the @typeclass annotation" should {
+    "not produce warnings" in withMrPlod("example.scala") { mr: MrPlod =>
+      mr.messages shouldBe 'empty
+
+      mr.typeAtPoint('def1).value shouldBe "T"
```

Comment:
fails with `<notype>`

Reply:
I'm not familiar with PC or pcplod so this may be a silly question. Does `def1` refer to only the `append` method on the `Semigroup` trait or also to the generated `append` method in the `Ops` trait? If the former, I'm not sure why simulacrum would be breaking anything as it does not rewrite signatures of type class trait methods.